src/TodoListItem.jsx

```diff
@@ -3,14 +3,7 @@ import React from 'react';
 function TodoListItem({ item }) {
 const myArray = ['apple', 'banana', 'orange'];
 
-  return (
-    <ul>
-      {item.text}
-      <li key={item.id}>{item.title}</li>
-      const myArray = ['apple', 'banana', 'orange'];
-
-    </ul>
-  );
+  return ;
```

Comment:
Hey Mohammad, it looks like you need to display the todos here in the return statement using data that you passed as a prop in each <TodoListItem /> component in TodoList.jsx. 

In TodoList.jsx, you should see that you're passing in a `key` and an `item` prop that should then be displayed in this component. Perhaps something like:

```tsx
return (
  <li>{item.title}</li>
)
```